risc0/build/src/config.rs

```diff
@@ -32,6 +34,9 @@ pub struct DockerOptions {
 
     /// Additional environment variables for the build container.
     pub env: Vec<(String, String)>,
+
+    /// Docker container tag to use
```

Comment:
do you need to add an attribute for the `Builder` derive?

Reply:
thanks for catching this